fdbserver/workloads/MemoryLifetime.actor.cpp

```diff
@@ -70,30 +70,32 @@ struct MemoryLifetime : KVWorkload {
 	ACTOR Future<Void> _start(Database cx, MemoryLifetime* self) {
 		state double startTime = now();
 		state ReadYourWritesTransaction tr(cx);
+		state Reverse reverse = Reverse::FALSE;
+		state Snapshot snapshot = Snapshot::FALSE;
 		loop {
 			try {
 				int op = deterministicRandom()->randomInt(0, 4);
 				if (op == 0) {
-					state bool getRange_isReverse = deterministicRandom()->random01() < 0.5;
+					reverse = Reverse{ deterministicRandom()->coinflip() };
 					state Key getRange_startKey = self->getRandomKey();
 					state KeyRange getRange_queryRange =
-					    getRange_isReverse ? KeyRangeRef(normalKeys.begin, keyAfter(getRange_startKey))
-					                       : KeyRangeRef(getRange_startKey, normalKeys.end);
+					    reverse ? KeyRangeRef(normalKeys.begin, keyAfter(getRange_startKey))
+					            : KeyRangeRef(getRange_startKey, normalKeys.end);
 					state bool getRange_randomStart = deterministicRandom()->random01();
 					state Value getRange_newValue = self->randomValue();
-					state bool getRange_isSnapshot = deterministicRandom()->random01() < 0.5;
+					snapshot = Snapshot{ deterministicRandom()->coinflip() };
```

Comment:
Is the right hand side type required for `operator =` assignments like this?  I made a similar comment for a state variable but the context could be different since state variable declarations with `=` become constructions if they are at the top of an actor.

Reply:
I've now added a `set` method to simplify cases like this.